benchmark/Graphs.jl

```diff
@@ -121,76 +144,56 @@ end
 
 g = path_graph(SymmetricGraph, n)
 lg = LG.Graph(g)
-bench["iter-edges"] = @benchmarkable bench_iter_edges($g)
-bench["iter-edges-vectorized"] = @benchmarkable bench_iter_edges_vectorized($g)
-bench["iter-edges-lightgraphs"] = @benchmarkable bench_iter_edges($lg)
-bench["has-edge"] = @benchmarkable bench_has_edge($g)
-bench["has-edge-lightgraphs"] = @benchmarkable bench_has_edge($lg)
-bench["iter-neighbors"] = @benchmarkable bench_iter_neighbors($g)
-bench["iter-neighbors-lightgraphs"] = @benchmarkable bench_iter_neighbors($lg)
 
-function lg_connected_components_projection(g)
-  label = Vector{Int}(undef, LG.nv(g))
-  LG.connected_components!(label, g)
-end
-
-n₀ = 2000
-g₀ = path_graph(SymmetricGraph, n₀)
-g = ob(coproduct(fill(g₀, 5)))
-lg = LG.Graph(g)
-bench["path-graph-components"] = @benchmarkable connected_components($g)
-bench["path-graph-components-proj"] =
-  @benchmarkable connected_component_projection($g)
-bench["path-graph-components-lightgraphs"] =
-  @benchmarkable LG.connected_components($lg)
-bench["path-graph-components-proj-lightgraphs"] =
-  @benchmarkable lg_connected_components_projection($lg)
+clbench["iter-edges"] = @benchmarkable bench_iter_edges($g)
+clvecbench["iter-edges"] = @benchmarkable bench_iter_edges_vectorized($g)
+lgbench["iter-edges"] = @benchmarkable bench_iter_edges($lg)
+clbench["has-edge"] = @benchmarkable bench_has_edge($g)
+lgbench["has-edge"] = @benchmarkable bench_has_edge($lg)
+clbench["iter-neighbors"] = @benchmarkable bench_iter_neighbors($g)
+lgbench["iter-neighbors"] = @benchmarkable bench_iter_neighbors($lg)
 
-g₀ = star_graph(SymmetricGraph, n₀)
-g = ob(coproduct(fill(g₀, 5)))
-lg = LG.Graph(g)
-bench["star-graph-components"] = @benchmarkable connected_components($g)
-bench["star-graph-components-proj"] =
-  @benchmarkable connected_component_projection($g)
-bench["star-graph-components-lightgraphs"] =
-  @benchmarkable LG.connected_components($lg)
-bench["star-graph-components-proj-lightgraphs"] =
-  @benchmarkable lg_connected_components_projection($lg)
+bench = SUITE["SymmetricGraphConnComponent"] = BenchmarkGroup()
+clbench = bench["Catlab"] = BenchmarkGroup()
+lgbench = bench["LightGraphs"] = BenchmarkGroup()
```

Comment:
Are the connected components benchmarks for symmetric graphs gone now?

Reply:
I think you may be misunderstanding this code structure? The symmetric graph connected component benchmarks for Catlab go in this `clbench`, and the ones for LightGraphs go in this `lgbench`.

Reply:
But I don't see any actual benchmarks for symmetric case, i.e., no connected components of symmetric graphs are computed? Compare the sections for graphs and symmetric graphs.